test/flows/child_benefit_tax_calculator_flow_test.rb

```diff
@@ -0,0 +1,399 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class ChildBenefitTaxCalculatorFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow ChildBenefitTaxCalculatorFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: how_many_children?" do
+    setup { testing_node :how_many_children? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "validation" do
+      should "be invalid for a number less than 1" do
+        assert_invalid_response "0"
+      end
+
+      should "be invalid for a number greater than 30" do
```

Comment:
Unrelated to your changes but... https://www.guinnessworldrecords.com/world-records/most-prolific-mother-ever

Reply:
Haha wow, that is incredible